go/ephemeral/user_ek_box_storage_test.go

```diff
@@ -38,17 +38,17 @@ func TestUserEKBoxStorage(t *testing.T) {
 	verifyUserEK(t, userEKMetadata, userEK)
 
 	// Test MaxGeneration
-	maxGeneration, err := s.MaxGeneration(context.Background())
```

Comment:
Before the force sync no EKs were generated on login so we had to change some assertions